pkg/types/aws/validation/machinepool.go

```diff
@@ -108,6 +109,25 @@ func validateIOPS(p *aws.MachinePool, fldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
+func validateThroughput(p *aws.MachinePool, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	volumeType := strings.ToLower(p.EC2RootVolume.Type)
+	throughput := p.EC2RootVolume.Throughput
+
+	switch volumeType {
+	case "gp3":
```

Comment:
@patrickdillon @jhixson74 should we allow `io2` too as it is commonly used to environments which requires highly throughput/performance on disks?

Reply:
https://docs.aws.amazon.com/cli/latest/reference/ec2/create-volume.html

Scroll down to --throughput

Only gp3 is supported.